src/effects/VST/VSTEffect.h

```diff
@@ -101,6 +101,17 @@ struct VSTEffectSettings
 
 struct VSTEffectWrapper : public VSTEffectLink, public XMLTagHandler
 {
+   VSTEffectWrapper(const PluginPath& path)
+      : mPath(path)
+   {}
+
+   // This constructor and SetPath are meant to be used when creating a VSTEffectInstance
+   VSTEffectWrapper() {}
+   void SetPath(const PluginPath& path)
```

Comment:
Ah, one more comment.  Honestly the last.

I don't think this function is necessary.

Reply:
That's right, I could add the PluginPath as an arg to the Instance constructor. Done in commit 4.4